docs/manifest.json

```diff
@@ -1582,5 +1576,11 @@
 		"slug": "packages-wordcount",
 		"markdown_source": "../packages/wordcount/README.md",
 		"parent": "packages"
+	},
+	{
+		"title": "@wordpress/wp-env",
```

Comment:
The tool that automatically generates this is incorrectly inserting the `@wordpress/` prefix.

Reply:
Is it hard to fix? I tend to disagree with my previous comment to keep the package that is an alias outside of Gutenberg 😂